src/api/orders/types.ts

```diff
@@ -56,3 +56,29 @@ export type OrderStatusResponse = {
     initialRateBump: number
     isNativeCurrency: boolean
 }
+
+export type OrdersByMakerParams = {
+    address: string
+} & OrdersByMakerQueryParams
+
+export type OrdersByMakerQueryParams = {
+    page?: number
+    limit?: number
+}
```

Comment:
This type can be moved nearby pagination types. And pagination you can wrap in the folder with `types.ts` file